### PR TITLE
added Phoenix_EV_Charger

### PR DESCRIPTION
**Pull request by abrlox**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @telicbrag, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#385).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/phoenix_ev_charger.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Phoenix EV Charger",
+  "owner": ["@abrlox"],
+  "manifest": "https://raw.githubusercontent.com/abrlox/home-assistant-phoenix-ev-charger/main/custom_components/phoenix_ev_charger/manifest.json",
```
Comment (ludeeus) [inline components/phoenix_ev_charger.json:4]: Your `pymodbus` requirement is conflicting with the one used in HA Core
<https://github.com/home-assistant/core/blob/dev/requirements_all.txt#L1557>
Please use `pymodbus>=2.5.1`

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: components/phoenix_ev_charger.json (hunk 1 above)
Comment (abrlox) [reply components/phoenix_ev_charger.json:4]: Hi ludeeus,

Thanks for the help.
I changed the requirement as requested. Sorry I didn't notice earlier.

**Review by frenck — APPROVED** [review]
Thanks, @abrlox 👍

**Pull request merged**